controllers/provider-azure/docs/migrate-loadbalancer.md

```diff
@@ -0,0 +1,68 @@
+# Migrate Azure Shoot LoadBalancer from basic to standard SKU
+
+This guide descibes how to migrate the LoadBalancer of a Azure Shoot cluster from the basic SKU to the standard SKU.<br/>
+**Be aware:** You need to delete and recreate all services of type LoadBalancer, which means that the public ip addresses of your service endpoints will change.<br/>
+Please do this only if you really need to migrate this Shoot to use standard LoadBalancers. All new Shoot cluster will automatically use standard LoadBalances.
+
+1. Disable temporarily Gardeners reconciliation.
+```sh
+# In the Garden cluster.
+kubectl annotate shoot <shoot-name> shoot.garden.sapcloud.io/ignore="true"
```

Comment:
This only works if it is enabled in the GCM config.

Reply:
Hmm ok. Wasn't aware of that. To do the migration successfully it is required to avoid any reconciliation during the process. I will add a note which explains that the GCM needs to be configured to allow ignoring Shoots. Thanks!

Reply:
done.